--- Makefile.orig	2005-01-20 15:09:26.000000000 +0100
+++ Makefile	2005-04-21 20:20:37.478680392 +0200
@@ -39,8 +39,8 @@ jpeg.o: jpeg.S
 	as -ahlsn=jpeg.lst -o $@ $<
 
 install: all
-	install -d -m 755 $(DESTDIR)/usr/sbin $(DESTDIR)/usr/share/gfxboot
-	install -m 755 mkbootmsg getx11font help2txt $(DESTDIR)/usr/sbin
+	install -d -m 755 $(DESTDIR)/usr/bin $(DESTDIR)/usr/share/gfxboot
+	install -m 755 mkbootmsg getx11font help2txt $(DESTDIR)/usr/bin
 #	install -m 644 $(LIBFILES) $(DESTDIR)/usr/share/gfxboot
 	cp -a themes $(DESTDIR)/usr/share/gfxboot
 
