--- Makefile.orig	2005-08-11 13:05:50.000000000 +0200
+++ Makefile	2005-08-11 13:10:06.453794250 +0200
@@ -6,7 +6,7 @@ THEMES	 = $(wildcard themes/*)
 
 .PHONY: all themes clean install doc
 
-all:	bin2c mkbootmsg bincode getx11font addblack
+all:	bin2c mkbootmsg bincode getx11font addblack doc
 
 getx11font: getx11font.c
 	$(CC) $(CFLAGS) -L$(X11LIBS) $< -lX11 -o $@
--- doc/Makefile.orig	2005-08-11 13:25:48.340658500 +0200
+++ doc/Makefile	2005-08-11 13:26:03.469604000 +0200
@@ -19,3 +19,7 @@ html: $(XMLSRC)
 
 clean:
 	rm -rf *~ *.pdf *.html *.txt reference.xml *.dsl *.local
+
+install: all
+	install -m 755 -d $(DESTDIR)/usr/share/doc/gfxboot
+	install -m 644    gfxboot.html $(DESTDIR)/usr/share/doc/gfxboot/
